docker/nginx.conf

```diff
@@ -1,72 +1,80 @@
-worker_processes 1;
```

Comment:
I'd suggest setting `worker_processes auto`? the default is 1 which isn't great

Reply:
The default in Docker is `auto`, which is why this line is removed.